CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Please see [doc/docs/project/CHANGELOG.md
```

Comment:
I'm still not a fan of the deep hierarchy, I don't think we have so many files that it makes sense.